src/server/debug/server/server.go

```diff
@@ -292,7 +292,16 @@ func (s *debugServer) makeTasks(ctx context.Context, request *debug.DumpV2Reques
 			}
 		}
 		if len(sys.Profiles) > 0 {
-			ts = append(ts, s.makeProfilesTask(server, sys.Profiles))
+			podCount := 0
```

Comment:
I think we typically prefer `var podCount int`.

The Google Style Guide doesn't make a decision, though:
![Capture](https://github.com/pachyderm/pachyderm/assets/2367/1fa57317-8d83-420d-ab45-b76b86da7609)

https://google.github.io/styleguide/go/decisions#non-decisions

I prefer the `var` form because it's consistent with slice initialization:

https://google.github.io/styleguide/go/decisions#nil-slices